src/index.js

```diff
@@ -488,9 +493,37 @@ class EventsCoreAPI {
    * @param {string} signatureHeaderValue Value of x-adobe-signature header in each POST request to the registered webhook URL
    * @returns {boolean} If signature matches return true else return false
    */
-  verifySignatureForEvent (event, clientSecret, signatureHeaderValue) {
```

Comment:
Wondering if we should keep this and mark it as deprecated instead of simply removing it in the next version. We should give our clients some time to migrate away from the SDK as well.

Reply:
okay